### PR TITLE
Add documentation for using accessible-menus.js

### PR DESCRIPTION
**Pull request by cvalarida**
Part 1 of https://github.com/department-of-veterans-affairs/vets.gov-team/issues/6433

Hopefully it's clear enough, but as always, I'm wide open to suggestions on how to make it more useful.

### CODE REVIEW

File: docs/AccessibleMenus.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+  - Left / right arrow keys
+  - With wraparound
+2. Open sub menus
+  - Down arrow / space / enter
+  - Focus should go to the first visible menu item in the sub-menu
+3. Activate menubar menu items
+  - Follow links with space / enter
+
+*Menus*
+The helper function is pretty naive right now, but it can be modified to be smarter. Currently, it assumes menus are vertical and sub-menus are opened to the right.
```
Comment (cehsu) [inline docs/AccessibleMenus.md]: Would it be of interest to create ticket for planned optimizations/list them here?

**Review by cehsu — APPROVED** [review]
This is an excellent tool for improving visibility/consensus on how we achieve accessibility.

**Review by jbalboni — APPROVED** [review]

**Review by ncksllvn — APPROVED** [review]
LGTM!

**Review by markgreenburg — APPROVED** [review]

File: docs/AccessibleMenus.md (hunk 1 above)
Comment (cvalarida) [reply docs/AccessibleMenus.md]: Mmm...I'm not sure, actually. For this particular enhancement, I think just mentioning this assumption is enough for now. For others... :man_shrugging: Probably a case-by-case basis.

I think planning for work is best recorded in separate tickets, though, not documentation.

**Pull request merged**